### PR TITLE
Add FLYWOOH743PRO

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: configs/FLYWOOH743PRO/config.h [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    TIMER_PIN_MAP( 3, MOTOR4_PIN, 2,  3) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN, 2,  4) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN, 2,  5) \
+    TIMER_PIN_MAP( 6, MOTOR7_PIN, 1,  6) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN, 1,  7) \
+    TIMER_PIN_MAP( 8, MOTOR9_PIN, 1, 12) \
+    TIMER_PIN_MAP( 9, MOTOR10_PIN, 1, -1) \
+    TIMER_PIN_MAP(10, SERVO1_PIN, 1,  0) \
+    TIMER_PIN_MAP(11, SERVO2_PIN, 1, -1) \
+    TIMER_PIN_MAP(12, LED_STRIP_PIN, 1, 14)
```
Comment (ot0tot) [inline configs/FLYWOOH743PRO/config.h:116]:
```suggestion
#define TIMER_PIN_MAPPING \
    TIMER_PIN_MAP( 0, MOTOR1_PIN ,  2,   0) \
    TIMER_PIN_MAP( 1, MOTOR2_PIN ,  2,   0) \
    TIMER_PIN_MAP( 2, MOTOR3_PIN ,  2,   0) \
    TIMER_PIN_MAP( 3, MOTOR4_PIN ,  2,   0) \
    TIMER_PIN_MAP( 4, MOTOR5_PIN ,  2,   0) \
    TIMER_PIN_MAP( 5, MOTOR6_PIN ,  2,   0) \
    TIMER_PIN_MAP( 6, MOTOR7_PIN ,  1,   0) \
    TIMER_PIN_MAP( 7, MOTOR8_PIN ,  1,   0) \
    TIMER_PIN_MAP( 8, MOTOR9_PIN ,  1,   0) \
    TIMER_PIN_MAP( 9, MOTOR10_PIN,  1,  0) \
    TIMER_PIN_MAP(10, SERVO1_PIN  ,  1, -1) \
    TIMER_PIN_MAP(11, SERVO2_PIN ,   1, -1) \
    TIMER_PIN_MAP(12, LED_STRIP_PIN, 1, 0) \
    TIMER_PIN_MAP(13, RX_PPM_PIN ,   2, -1)
```

**Review by ot0tot — APPROVED** [review]

File: configs/FLYWOOH743PRO/config.h [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    TIMER_PIN_MAP( 6, MOTOR7_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 8, MOTOR9_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 9, MOTOR10_PIN,   1,   0) \
+    TIMER_PIN_MAP(10, SERVO1_PIN ,   1,  -1) \
+    TIMER_PIN_MAP(11, SERVO2_PIN ,   1,  -1) \
+    TIMER_PIN_MAP(12, LED_STRIP_PIN, 1,   0) \
+    TIMER_PIN_MAP(13, RX_PPM_PIN ,   2,  -1)
+
+#define ADC1_DMA_OPT        8
```
Comment (sugaarK) [inline configs/FLYWOOH743PRO/config.h:119]: do these need defining with the DMA controller?

Comment (KarateBrot) [reply configs/FLYWOOH743PRO/config.h:119]: Looks like this is just copied over from another target. I did that as well for a H743 config but turned out it was wrong.

File: configs/FLYWOOH743PRO/config.h [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    TIMER_PIN_MAP( 0, MOTOR1_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN ,   2,   0) \
+    TIMER_PIN_MAP( 6, MOTOR7_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 8, MOTOR9_PIN ,   1,   0) \
+    TIMER_PIN_MAP( 9, MOTOR10_PIN,   1,   0) \
```
Comment (KarateBrot) [inline configs/FLYWOOH743PRO/config.h:113]: Are you sure this is correct? We only support 8 motors

File: configs/FLYWOOH743PRO/config.h (hunk 2 above)
Comment (KarateBrot) [reply configs/FLYWOOH743PRO/config.h:119]: Looks like this is just copied over from another target. I did that as well for a H743 config but turned out it was wrong.

File: configs/FLYWOOH743PRO/config.h (hunk 3 above)
Comment (haslinghuis) [reply configs/FLYWOOH743PRO/config.h]: Sorted.

```
Currently active Timers:
-----------------------
TIM1:
    CH1 : LED_STRIP
TIM2: FREE
TIM3: FREE
TIM4:
    CH3 : SERVO 1
    CH4 : SERVO 2
TIM5: FREE
TIM6: FREE
TIM7: FREE
TIM8:
    CH2 : DSHOT_BITBANG 2
    CH3 : DSHOT_BITBANG 1
TIM12: FREE
TIM13: FREE
TIM14: FREE
TIM15:
    CH1 : SERVO 3
    CH2 : SERVO 4
TIM16: FREE
TIM17: FREE

Currently active DMA:
--------------------
DMA1 Stream 0: LED_STRIP
DMA1 Stream 1: DSHOT_BITBANG 2
DMA1 Stream 2: DSHOT_BITBANG 1
DMA1 Stream 3: SPI_SDO 1
DMA1 Stream 4: SPI_SDI 1
DMA1 Stream 5: SPI_SDO 4
DMA1 Stream 6: SPI_SDI 4
DMA1 Stream 7: FREE
DMA2 Stream 0: ADC 1
DMA2 Stream 1: ADC 3
DMA2 Stream 2: FREE
DMA2 Stream 3: FREE
DMA2 Stream 4: FREE
DMA2 Stream 5: FREE
DMA2 Stream 6: FREE
DMA2 Stream 7: FREE
```

File: configs/FLYWOOH743PRO/config.h (hunk 2 above)
Comment (haslinghuis) [reply configs/FLYWOOH743PRO/config.h:119]: This is very common setting:

```
src/config/configs/SPRACINGH7EF/config.h:162:#define ADC3_DMA_OPT                  10
src/config/configs/FOXEERH743V2/config.h:107:#define ADC1_DMA_OPT 8
src/config/configs/FOXEERH743V2/config.h:108:#define ADC2_DMA_OPT 9
src/config/configs/FOXEERH743V2/config.h:109:#define ADC3_DMA_OPT 10
src/config/configs/IFLIGHT_H743_AIO/config.h:92:#define ADC1_DMA_OPT 8
src/config/configs/IFLIGHT_H743_AIO/config.h:93:#define ADC3_DMA_OPT 9
src/config/configs/SPRACINGH7EVO/config.h:122:#define ADC3_DMA_OPT                  10
src/config/configs/MINI_H743_HD/config.h:119:#define ADC1_DMA_OPT        8
src/config/configs/MINI_H743_HD/config.h:120:#define ADC2_DMA_OPT        9
src/config/configs/MINI_H743_HD/config.h:121:#define ADC3_DMA_OPT       10
src/config/configs/NEUTRONRCH7BT/config.h:121:#define ADC1_DMA_OPT        8
src/config/configs/NEUTRONRCH7BT/config.h:122:#define ADC3_DMA_OPT        9
src/config/configs/IFLIGHT_H7_TWING/config.h:103:#define ADC1_DMA_OPT        8
src/config/configs/IFLIGHT_H7_TWING/config.h:104:#define ADC3_DMA_OPT        9
src/config/configs/JHEH743_HD/config.h:123:#define ADC1_DMA_OPT 8
src/config/configs/JHEH743_HD/config.h:124:#define ADC3_DMA_OPT 9
src/config/configs/MAMBAH743/config.h:114:#define ADC1_DMA_OPT        8
src/config/configs/MAMBAH743/config.h:115:#define ADC3_DMA_OPT        9
src/config/configs/SDMODELH7V2/config.h:105:#define ADC1_DMA_OPT        8
src/config/configs/SDMODELH7V2/config.h:106:#define ADC3_DMA_OPT        9
src/config/configs/MATEKH743/config.h:138:#define ADC1_DMA_OPT        8
src/config/configs/MATEKH743/config.h:139:#define ADC3_DMA_OPT        9
src/config/configs/SPRACINGH7NANO/config.h:106:#define ADC1_DMA_OPT            8
src/config/configs/SPRACINGH7NANO/config.h:107:#define ADC3_DMA_OPT            9
src/config/configs/IFLIGHT_H743_AIO_V2/config.h:106:#define ADC1_DMA_OPT        1
src/config/configs/IFLIGHT_H743_AIO_V2/config.h:107:#define ADC2_DMA_OPT        9
src/config/configs/IFLIGHT_H743_AIO_V2/config.h:108:#define ADC3_DMA_OPT       10
src/config/configs/SPRACINGH7EXTREME/config.h:111:#define ADC1_DMA_OPT 8
src/config/configs/SPRACINGH7EXTREME/config.h:112:#define ADC3_DMA_OPT 9
src/config/configs/SPRACINGH7EXTREME/config.h:226:#define ADC1_DMA_OPT        8
src/config/configs/SPRACINGH7EXTREME/config.h:227:#define ADC3_DMA_OPT        9
src/config/configs/TMOTORH743/config.h:122:#define ADC1_DMA_OPT                    8
src/config/configs/TMOTORH743/config.h:123:#define ADC3_DMA_OPT                    9
src/config/configs/KAKUTEH7/config.h:100:#define ADC1_DMA_OPT        8
src/config/configs/KAKUTEH7/config.h:101:#define ADC3_DMA_OPT        9
src/config/configs/KAKUTEH7V2/config.h:96:#define ADC1_DMA_OPT        8
src/config/configs/KAKUTEH7V2/config.h:97:#define ADC3_DMA_OPT        9
src/config/configs/SEQUREH7/config.h:99:#define ADC1_DMA_OPT 8
src/config/configs/SEQUREH7/config.h:100:#define ADC3_DMA_OPT 9
src/config/configs/MAMBAH743_2022B/config.h:114:#define ADC1_DMA_OPT        8
src/config/configs/MAMBAH743_2022B/config.h:115:#define ADC3_DMA_OPT        9
src/config/configs/SPRACINGH7ZERO/config.h:119:#define ADC1_DMA_OPT            8
src/config/configs/SPRACINGH7ZERO/config.h:120:#define ADC3_DMA_OPT            9
src/config/configs/KAKUTEH7MINI/config.h:100:#define ADC1_DMA_OPT        8
src/config/configs/KAKUTEH7MINI/config.h:101:#define ADC3_DMA_OPT        9
src/config/configs/SPRACINGH7RF/config.h:125:#define ADC3_DMA_OPT                  10
src/config/configs/SKYSTARSH7HD/config.h:123:#define ADC1_DMA_OPT        8
src/config/configs/SKYSTARSH7HD/config.h:124:#define ADC3_DMA_OPT        9
src/config/configs/SEQUREH7V2/config.h:100:#define ADC1_DMA_OPT 8
src/config/configs/SEQUREH7V2/config.h:101:#define ADC3_DMA_OPT 9
src/config/configs/FLYWOOH743PRO/config.h:119:#define ADC1_DMA_OPT        8
src/config/configs/FLYWOOH743PRO/config.h:120:#define ADC3_DMA_OPT        9
src/config/configs/SDMODELH7/config.h:98:#define ADC1_DMA_OPT 8
src/config/configs/SDMODELH7/config.h:99:#define ADC3_DMA_OPT 9
src/config/configs/SAKURAH743/config.h:157:#define ADC1_DMA_OPT        8
src/config/configs/SAKURAH743/config.h:158:#define ADC3_DMA_OPT        9
src/config/configs/CHONKERH735/config.h:109:#define ADC1_DMA_OPT                    8
src/config/configs/CHONKERH735/config.h:110:#define ADC3_DMA_OPT                    9
src/config/configs/NEUTRONRCH743AIO/config.h:116:#define ADC1_DMA_OPT        8
src/config/configs/NEUTRONRCH743AIO/config.h:117:#define ADC3_DMA_OPT        9
src/config/configs/2RAW_THUNDER_H7/config.h:112:#define ADC1_DMA_OPT        8
src/config/configs/2RAW_THUNDER_H7/config.h:113:#define ADC3_DMA_OPT        9
src/config/configs/TMH7/config.h:107:#define ADC1_DMA_OPT        8
src/config/configs/TMH7/config.h:108:#define ADC3_DMA_OPT        9
src/config/configs/IFLIGHT_BLITZ_H7_PRO/config.h:121:#define ADC1_DMA_OPT        8
src/config/configs/IFLIGHT_BLITZ_H7_PRO/config.h:122:#define ADC3_DMA_OPT        9
src/config/configs/BETAFPVH743/config.h:111:#define ADC1_DMA_OPT        1
src/config/configs/BETAFPVH743/config.h:112:#define ADC2_DMA_OPT        9
src/config/configs/BETAFPVH743/config.h:113:#define ADC3_DMA_OPT       10
src/config/configs/JHEH743_AIO/config.h:107:#define ADC1_DMA_OPT        8
src/config/configs/JHEH743_AIO/config.h:108:#define ADC3_DMA_OPT        9
src/config/configs/FOXEERH743/config.h:105:#define ADC1_DMA_OPT                    8
src/config/configs/FOXEERH743/config.h:106:#define ADC2_DMA_OPT                    9
src/config/configs/FOXEERH743/config.h:107:#define ADC3_DMA_OPT                    10
src/config/configs/AOCODARCH7DUAL/config.h:122:#define ADC1_DMA_OPT        8
src/config/configs/AOCODARCH7DUAL/config.h:123:#define ADC3_DMA_OPT        9
```

Comment (haslinghuis) [reply configs/FLYWOOH743PRO/config.h:119]: Wonder if `IFLIGHT_H743_AIO_V2` and `BETAFPVH743` seems to be wrong here.

**Review by blckmn — APPROVED** [review]

File: configs/FLYWOOH743PRO/config.h (hunk 2 above)
Comment (sugaarK) [reply configs/FLYWOOH743PRO/config.h:119]: Doesn’t answer my question.. with the dma controller it’s better to let it assign dma channels

**Pull request merged**